hazelcast/src/main/java/com/hazelcast/map/impl/proxy/NearCachedMapProxyImpl.java

```diff
@@ -73,6 +75,8 @@
     public NearCachedMapProxyImpl(String name, MapService mapService, NodeEngine nodeEngine, MapConfig mapConfig) {
         super(name, mapService, nodeEngine, mapConfig);
 
+        localMember = getNodeEngine().getLocalMember();
```

Comment:
You should not cache localMember. When lite member is promoted, `localMember` field of ClusterService is updated.